assets/js/src/index.js

```diff
@@ -15,7 +15,33 @@ if ( window.ga4w ) {
 		window.addEventListener( 'load', warnIfDataMissing );
 	}
 }
+
+const consentMap = {
+	statistics: [ 'analytics_storage' ],
+	marketing: [ 'ad_storage', 'ad_user_data', 'ad_personalization' ],
+};
+
 function initializeTracking() {
+	// eslint-disable-next-line camelcase
```

Comment:
:nail_care: 
WDYT of explaining why we disable this rule:
```suggestion
	// eslint-disable-next-line camelcase -- `wp_has_consent` is defined by the WP Consent API plugin.
```

Reply:
Added in 10c2af7